client/app/bundles/course/assessment/submission/reducers/questionsFlags.js

```diff
@@ -71,6 +72,7 @@ export default function (state = {}, action) {
         },
       };
     }
+    case actions.UNSUBMIT_SUCCESS:
```

Comment:
This line is unreachable, because the same `case` is above. Can I check which one is the one you want?